index.html

```diff
@@ -4,7 +4,6 @@
     <meta charset="UTF-8" />
     <link rel="icon" href="favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0" />
-    <script src="config.js"></script>
```

Comment:
![image](https://github.com/kbss-cvut/record-manager-ui/assets/3962237/b5d82d36-dc38-49c4-88a6-798d499d5c87)

I did change as indicated above to fix build warning ... so rollup `htmlPlugin` might not be needed. Is there any catch to that solution?
I tested the solution in `deploy/internal-auth` at it works for me.

@shellyear Could you please check if it is viable solution?